lib/net/dns/resolver.rb

```diff
@@ -1093,7 +1093,7 @@ def parse_config_file
             when /^\s*search\s+(.*)/
               self.searchlist = $1.split(" ")
             when /^\s*nameserver\s+(.*)/
-              self.nameservers = $1.split(" ")
+              self.nameservers += $1.split(" ")
             end
           end
         end
```

Comment:
Does the windows codepath get handled correctly? Probably worth a check after the unix-y path works 👀

Reply:
according to this random issue report (I couldn't find a better example of what gets returned) it _should_ be correct for windows
https://bugs.ruby-lang.org/issues/12604

but I'll be sure to verify this on windows myself too

Reply:
I have a windows dev env set up too, let me know if you've got replication steps you want me to follow :+1: